README.md

```diff
@@ -30,3 +30,61 @@ Test the connectivity with the Cups server using lpstat -r
 
 Test that printers are detected using lpstat -v
 
+## Using the APIs
+
+### V2 API
+
+Support for both Toshiba and Squix printers. When it receives a request, PMB looks at the printer name it is passed in the request, and determines whether it's a Toshiba or a Squix (`printers` database table, `printer_type` column). It then handles the printing itself for Toshibas, or sends a request to SPrint (https://github.com/sanger/sprint) for Squixes.
+The same templates (`label_templates` and `label_types` tables) can be used with both types of printer.
+
+Print job POST request bodies will look something like this:
+
+```
```

Comment:
Is it also worth adding something about the headers? Not sure if that is explained elsewhere.

Reply:
I'm not really sure which headers are required? I'll merge this in as-is, and we can add extra info as needed in future (or now if you know the answer off the top of your head?).